backend/models/issue.js

```diff
@@ -1219,6 +1225,14 @@ schema.statics.isIssueBeingClosed = function(oldIssue, newIssue) {
 	return oldIssue.status !== "closed" &&  newIssue.status === "closed";
 };
 
+schema.statics.isIssueBeingReopened = function (oldIssue, newIssue) {
+	if (!oldIssue) {
+		return false;
+	}
+
+	return oldIssue.status === "closed" && newIssue.status !== "closed";
```

Comment:
remove the early reject and merge it into one boolean operation
```suggestion
	return oldIssue && oldIssue.status === "closed" && newIssue.status !== "closed";
```